### DISCUSSION TITLE
Invoke-PnPTenantTemplate is not performing any action, and ends after 2 seconds

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [pnp/powershell#4099] [Q&A]
I have 2 SharePoint Online tenants which both have a modern communication sites. now i used this command to get the source site template:-

`PS C:\Users\mohan> Connect-PnPOnline -Url "https://ssms.sharepoint.com/sites/PPM-test/" -Interactive`

`PS C:\Users\mohan> Get-PnPSiteTemplate -Out "PnP-Provisioning-F.xml"`

and i got this file:-

![image](https://github.com/user-attachments/assets/2492b3c2-73ab-47b7-a4b7-85f9cbaf1a06)

but when i try to apply the site template to the destination site using this command:-

`PS C:\Users\mohan> Connect-PnPOnline -Url "https://destination.sharepoint.com/sites/PPM-test/" -Interactive`

`PS C:\Users\mohan> Invoke-PnPTenantTemplate -Path "PnP-Provisioning-F.xml"`

the command show the following text:-

"initializing engine....Applying Template... " 

then it ends after 2 seconds, without applying any thing to the destination.. so what can be causing this issue?

My Powershell version is PowerShell 7.4.2. 

Also when i run the `Invoke-PnPTenantTemplate using -Verbose`, i got this error `Access token acquired`:-

**PS C:\Users\mohan> Connect-PnPOnline -Url "https://destination.sharepoint.com/sites/PPM-test/" -Interactive
PS C:\Users\mohan> Invoke-PnPTenantTemplate -Path "PnP-Provisioning-F.xml" -Verbose
VERBOSE: Connecting to the SharePoint Online Admin Center at 'https://destination-admin.sharepoint.com' to run this cmdlet
VERBOSE: Connected to the SharePoint Online Admin Center at 'https://destination-admin.sharepoint.com' to run this cmdlet
VERBOSE: Acquiring oAuth token for permission scope Group.ReadWrite.All
VERBOSE: Access token acquired**

![image](https://github.com/user-attachments/assets/e2426785-caaf-4a6e-91be-e5ae5509cb44)

I also updated my Powershell version to PowerShell 7.4.3. but i got the exact same error . i am also the site admin and sharepoint admin on the destination

**Comment by rockraider213** [comment]
Hey Ive got similiar situation. I didnt test it with -Verbose. Maybe Ive git the same issue hidden. 

See: https://github.com/pnp/powershell/discussions/4098

**Reply by rockraider213** [reply]
also no errors with -Verbose strange thing
